src/main.rs

```diff
@@ -1,9 +1,8 @@
+use irc_tokio as irc;
```

Comment:
I'm not familiar with this syntax:  `use irc_tokio as irc;` what exactly does `irc_tokio` refer to here?

Reply:
`irc_tokio` is the name of the library (you can see in the `Cargo.toml`, it's usually just the name of the crate but you can customize in the config too).

The renaming of modules (using `as`) on import is usually used to prevent namespace collisions. For example, if importing different `Error` from multiple places, you could do `use std::error::Error as StdError` for one and `use irc_tokio::error::Error` for the other. Then no collision.

In this case, I renamed because everything in `main` referred to the `irc` module already, so it was easier than renaming each instance of `irc`.

I guess some people could also rename just because a module name is too long, or it's so frequently used it would create a lot of clutter.